index.html

```diff
@@ -2,8 +2,106 @@
 <html lang="en">
 <head>
     <title>James Cullen Portfolio</title>
+    <link rel="stylesheet" type="text/css" href= "styles.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 <body>
-
+    <header>
+        <nav>
+            <div>
+                <ul class="container nav-main">
+                    <li class="nav-buttons">
+                        <a href="#contact">
+                            <img src="nav-bar-icon-contact.png" alt="James Cullen Contact">
+                        </a>
+                    </li>
+                    <li class="nav-buttons">
+                        <a href="#portfolio">
+                            <img src="nav-bar-icon-portfolio.png" alt="James Cullen Portfolio">
+                        </a>
+                    </li>
+                    <li class="nav-buttons">
+                        <a href="#about">
+                            <img src="nav-bar-icon-about-me-link.png" alt="James Cullen About Me">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <main class="container">
+        <section id="about">
+            <article>
```

Comment:
could produce the same effect without the use of articles as they are not doing anything

Reply:
cool, I've taken them out